entry.c

```diff
@@ -218,10 +218,16 @@ load_entry(FILE *file, void (*error_func)(), struct passwd *pw, char **envp) {
 		/* DOM (days of month)
 		 */
 
-		if (ch == '*')
-			e->flags |= DOM_STAR;
-		ch = get_list(e->dom, FIRST_DOM, LAST_DOM,
-			      PPC_NULL, ch, file);
+		if (ch == '$') {
```

Comment:
should we also allow 'L' to indicate last day-of-month?

Reply:
no.